src/main/java/ch/njol/skript/aliases/ItemType.java

```diff
@@ -553,7 +553,6 @@ public ItemType clone() {
 	 * @see #removeFrom(ItemStack)
 	 * @see #removeFrom(List...)
 	 */
-	@Nullable
```

Comment:
I wonder why it was annotated as Nullable, the getStack method is NotNull so it is quite impossible to get null here. It was probably that way before the aliases rework and no one realized until now

Reply:
It seems to be leftover from https://github.com/SkriptLang/Skript/commit/91b28639c1cb1ba73de4a34b05df69ae3f4282df#diff-1578d77b4b4b435cfe11847dbe3451001a3ef28efca17d2ea12896cd5f36a5a7L527